packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -236,9 +236,11 @@ export async function createNewUsersConnectToOrgIfExists({
         const { orgId, autoAccept } = connectionInfoMap[usernameOrEmail];
         const [emailUser, emailDomain] = usernameOrEmail.split("@");
         const username =
-          emailDomain === autoAcceptEmailDomain
-            ? slugify(emailUser)
-            : slugify(`${emailUser}-${emailDomain.split(".")[0]}`);
+          input.isOrg || parentId
```

Comment:
Ensures that for a regular team the username isn't set because user is allowed to change the username during signup in that context.